_applications/s3-source-example.md

```diff
@@ -35,18 +35,20 @@ No architecture, this is a connectivity example.
 Start a Jupyter notebook with,
 
 ```
-oc new-app mattf/workshop:base
-oc expose svc/workshop
+oc new-app radanalyticsio/base-notebook \
+   -e JUPYTER_NOTEBOOK_PASSWORD=supersecret \
+   -e JUPYTER_NOTEBOOK_X_INCLUDE=https://raw.githubusercontent.com/radanalyticsio/radanalyticsio.github.io/master/assets/s3-source-example/s3-source-example.ipynb
```

Comment:
any reason not to use `https://radanalytics.io/assets/s3-source-example/s3-source-example.ipynb` ?